docs/concepts/index.rst

```diff
@@ -0,0 +1,263 @@
+         called *eigenstates*, to energies. Only when the system is in an
+         eigenstate of the Hamiltonian is its energy well defined and called
+         the *eigenenergy*. When the system is in any other state, its energy
+         is uncertain.
+         For the D-Wave system, the Hamiltonian may be represented as
+
+         .. math::
+         	:nowrap:
+
+         	\begin{equation}
+         			{\cal H}_{ising} = \underbrace{\frac{A({s})}{2} \left(\sum_i {\hat\sigma_{x}^{(i)}}\right)}_\text{Initial Hamiltonian} + \underbrace{\frac{B({s})}{2} \left(\sum_{i} h_i {\hat\sigma_{z}^{(i)}} + \sum_{i>j} J_{i,j} {\hat\sigma_{z}^{(i)}} {\hat\sigma_{z}^{(j)}}\right)}_\text{Final Hamiltonian}
+         	\end{equation}
+
+         where :math:`{\hat\sigma_{x,z}^{(i)}}` are Pauli matrices operating on
+         a qubit :math:`q_i`, and :math:`h_i` and :math:`J_{i,j}` are the qubit
+         biases and coupling strengths.
+
+      Hardware graph
+         See `hardware graph`_. The hardware graph is the physical lattice of
+         interconnected qubits. See also :term:`working graph`.
+         See a fuller description under :doc:`QPU Topology </concepts/topology>`.
+
+         .. _hardware graph: https://docs.dwavesys.com/docs/latest/c_gs_4.html
+
+      Ising
+         Traditionally used in statistical mechanics. Variables are "spin up"
+         (:math:`\uparrow`) and "spin down" (:math:`\downarrow`), states that
+         correspond to :math:`+1` and :math:`-1` values. Relationships between
+         the spins, represented by couplings, are correlations or anti-correlations.
+         The :term:`objective function` expressed as an Ising model is as follows:
+
+         .. math::
+	          :nowrap:
+
+	          \begin{equation}
+	               \text{E}_{ising}(\pmb{s}) = \sum_{i=1}^N h_i s_i + \sum_{i=1}^N \sum_{j=i+1}^N J_{i,j} s_i s_j
+	          \end{equation}
+
+         where the linear coefficients corresponding to qubit biases
+         are :math:`h_i`, and the quadratic coefficients corresponding to coupling
+         strengths are :math:`J_{i,j}`.
+
+      Minimum gap
+         The minimum distance between the ground state and the first excited
+         state throughout any point in the anneal.
+
+      Model
+         A collection of variables with associated linear and
+         quadratic biases. Sometimes referred to as a **problem**.
+
+      Objective function
+         A mathematical expression of the energy of a system as a function of
+         binary variables representing the qubits.
+
+      Pegasus
+         The D-Wave :term:`QPU` is a lattice of interconnected qubits. While some qubits
+         connect to others via couplers, the D-Wave QPU is not fully connected.
+         Instead, the qubits interconnect in an architecture known as Pegasus.
+         See a fuller description under :doc:`QPU Topology </concepts/topology>`.
+
+      Penalty function
```

Comment:
Might as well add `penalty model` as well, you'll need it later.

Reply:
Added